xml/System.Threading.Tasks.Dataflow/BatchedJoinBlock`2.xml

```diff
@@ -112,6 +112,7 @@
         <summary>Initializes a new <see cref="T:System.Threading.Tasks.Dataflow.BatchedJoinBlock`2" /> with the specified configuration.</summary>
         <remarks>To be added.</remarks>
         <exception cref="T:System.ArgumentNullException">The <paramref name="dataflowBlockOptions" /> is <see langword="null" />.</exception>
+        <exception cref="T:System.ArgumentOutOfRangeException">The <paramref name="batchSize" /> must be positive.</exception>
```

Comment:
We usually focus on the condition of the exception not on the fix. We could say is not positive as well.

```suggestion
        <exception cref="T:System.ArgumentOutOfRangeException"><paramref name="batchSize" /> is less than one.</exception>
```